runelite-client/src/main/java/net/runelite/client/plugins/wildernesslocations/WildernessLocationsPlugin.java

```diff
@@ -177,7 +177,7 @@ private void onGameTick(GameTick event)
 			|| (this.pvpWorld && WorldType.isAllPvpWorld(client.getWorldType())));
 		if (renderLocation)
 		{
-			if (client.getLocalPlayer().getWorldLocation() != worldPoint)
+			if (client.getLocalPlayer() != null && client.getLocalPlayer().getWorldLocation() != worldPoint)
```

Comment:
Seems like client.getlocalplayer is being called a lot, would it be worth it to do final Player player = client.getlocalplayer?

Reply:
If we're talking readability: yes. If we're talking efficiency: probably not. Jvm could probably do voodoo magic to make it just as fast